README.md

```diff
@@ -30,9 +30,20 @@ It is also used to scaffold contig pairs linked by ARCS/ARKS.
 - Perl (tested on v5.32.1)
 - Autotools (if cloning directly from repository)
 
-### Compilation:
+### Installation:
 -------------------------------
-If cloning directly from the repository run:
+#### Installing ntJoin using Brew
+LINKS can be installed using [Homebrew](https://brew.sh) on macOS or [Linuxbrew](http://linuxbrew.sh) on Linux:
```

Comment:
Have we updated LINKS in brew yet? If not, would be good to do, especially if we're mentioning it in the README (which makes sense!)

Reply:
Yes. Recipe is ready and checked on my side. I will create the PR on brew after merging this PR